px-context-browser.html

```diff
@@ -232,6 +326,27 @@
         reflectToAttribute: true
       },
 
+      /**
+       * Watch for changes to determine if the context browser has failed to sync
```

Comment:
The developer sets this, not watches it for changes, right? They decide when favoritedSyncFailed, and they decide when it succeeds. If so we should update the docs to say that.

Set to `true` if your app fails to sync the user's favorite list to the server. Shows a notification bar informing the user that the sync failed and allows them to tap to attempt to sync the favorite list again. Listen for the `NAME_OF_EVENT` event to receive a notification when the user requests a new sync attempt.

Reply:
:+1: